pkg/manifest/raw_bootc.go

```diff
@@ -148,16 +181,12 @@ func (p *RawBootcImage) serialize() osbuild.Pipeline {
 		groupsStage.Devices = devices
 		pipeline.AddStage(groupsStage)
 	}
+
 	if len(p.Users) > 0 {
-		// ensure /var/home is available
+		// ensure home root dir (currently /var/home, /var/roothome) is
+		// available
 		mkdirStage := osbuild.NewMkdirStage(&osbuild.MkdirStageOptions{
```

Comment:
Can we skip the entire `osbuild.NewMkdirStage` if `buildHomedirPaths(...) == nil`?

Reply:
Thanks, that is an interesting question. In practice `if len(p.Users) > 0` then there will always be a buildHomedirPath result that is also > 0 - but that is true today who knows about tomorrow so maybe worth adding (otoh, tradeoffs, it adds one more level of if nested that is currently not needed and even if we never update it the worst case would be that we have an mkdir stage that just does nothing) - sorry, not very helpful I suppose :)

Reply:
As long as the first premise holds (we always have to create homedirs for users) it's fine to keep as is :)